db/migrations/aggregator/002.sql

```diff
@@ -0,0 +1,8 @@
+-- +migrate Up
+DELETE FROM aggregator.batch;
```

Comment:
I think that is not expected to delete table batch

Reply:
@ToniRamirezM explain that is the expected behaviour. Fist delete all entries of table `batch` to be able to apply the nullable change. This table is filled in each start so the content can be discarded